### PR TITLE
[replay] add replay player

### PR DESCRIPTION
**Pull request by simonlingoogle**
This PR adds a new program for playing replay: `otns-replay` based on PR #76 

Example usage:
```
$ otns-replay otns_0.replay
```

### CODE REVIEW

File: cmd/otns-replay/grpc_service.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
+package main
```
Comment (wgtdkp) [inline cmd/otns-replay/grpc_service.go:1]: License header.

File: cmd/otns-replay/otns_replay.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
+package main
```
Comment (wgtdkp) [inline cmd/otns-replay/otns_replay.go:1]: dito.

File: otns_main/otns_main.go [hunk 3]
```diff
@@ -143,13 +143,14 @@ func Main(visualizerCreator func(ctx *progctx.ProgCtx, args *MainArgs) visualize
...
+	replayFn := fmt.Sprintf("otns_%s.replay", os.Getenv("PORT_OFFSET"))
 	if vis != nil {
 		vis = visualizeMulti.NewMultiVisualizer(
 			vis,
-			visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr),
+			visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr, "replayFn"),
 		)
 	} else {
-		vis = visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr)
+		vis = visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr, replayFn)
```
Comment (wgtdkp) [inline otns_main/otns_main.go:153]: What is the difference between
```go
visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr, "replayFn")
```
and
```go
visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr, replayFn)
```
?

Comment (simonlingoogle) [reply otns_main/otns_main.go:153]: It's good interview question.

Comment (simonlingoogle) [reply otns_main/otns_main.go:153]: fixed

File: cmd/otns-replay/otns_replay.go (hunk 2 above)
Comment (simonlingoogle) [reply cmd/otns-replay/otns_replay.go:27]: added

File: cmd/otns-replay/grpc_service.go (hunk 1 above)
Comment (simonlingoogle) [reply cmd/otns-replay/grpc_service.go:27]: added

**Comment by simonlingoogle** [comment]
> should we include file `web/site/bindata.go`?

This PR neither changes the web site, or the gRPC protocol, thus no `bindata.go` change.

**Review by wgtdkp — COMMENTED** [review]
LGTM :+1:

**Review by wgtdkp — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**